.goreleaser.yml

```diff
@@ -118,12 +123,21 @@ nfpms:
     homepage: https://secretless.io
     license: "Apache 2.0"
     maintainer: CyberArk Maintainers <conj_maintainers@cyberark.com>
-    name_template: "{{.ProjectName}}_{{.Version}}_{{.Arch}}"
+    file_name_template: "{{.ProjectName}}_{{.Version}}_{{.Arch}}"
     vendor: CyberArk
 
 snapshot:
   name_template: "{{ .Tag }}-next"
 
 release:
-  disable: true
-  prerelease: auto
+  github:
+    owner: conjurinc
+    name: playroom
```

Comment:
Both of these GH params are wrong. Also, I think this will try to auto-release the broker on GH too when we run goreleaser.

Reply:
Actually, this GH release should probably be completely removed afaict

Reply:
The params are wrong on purpose, for testing. The params won't be necessary on merge because it'll infer from the repo.

Reply:
maybe it's worth leaving this unresolved until these are removed prior to merging? otherwise we may accidentally merge with these in (which we don't want)